modules/goldilocks/namespace.tf

```diff
@@ -0,0 +1,9 @@
+resource "kubernetes_manifest" "create_namespace" {
```

Comment:
can you please chose more simple name for the resource? for example, just `namespace` or `this`